unittests/Interpreter/InterpreterInfra.cpp

```diff
@@ -223,6 +247,62 @@ bool KBTesting::nextKB(llvm::KnownBits &x) {
   return false;
 }
 
+bool testKB(llvm::KnownBits Calculated, EvalValueKB Expected,
+            Inst::Kind K, std::vector<llvm::KnownBits> KBS) {
+    // expected value is poison/ub; so let binary transfer functions do
```

Comment:
plz remove spurious space in front of //

Reply:
fixed